packages/now-client/src/deploy.ts

```diff
@@ -38,25 +38,36 @@ async function* createDeployment(
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
-          Authorization: `Bearer ${options.token}`
+          Authorization: `Bearer ${options.token}`,
         },
         body: JSON.stringify({
           ...metadata,
-          files: preparedFiles
-        })
+          files: preparedFiles,
+        }),
       }
-    )
+    );
 
-    const json = await dpl.json()
+    const json = await dpl.json();
 
     if (!dpl.ok || json.error) {
       // Return error object
-      return yield { type: 'error', payload: json.error ? { ...json.error, status: dpl.status } : { ...json, status: dpl.status } }
+      return yield {
+        type: 'error',
+        payload: json.error
+          ? { ...json.error, status: dpl.status }
+          : { ...json, status: dpl.status },
+      };
     }
 
-    yield { type: 'created', payload: json }
+    for (const [name, value] of dpl.headers.entries()) {
+      if (name.startsWith('x-now-warning-')) {
+        yield { type: 'warning', payload: value };
```

Comment:
Is this printed by Now CLI? I can't seem to find it

Reply:
Yes, every time there's a `warning` event from `now-client`, it logs it with `WARN`
https://github.com/zeit/now/blob/b097496c7bf271db5dc63c13183e71760e09b68d/packages/now-cli/src/util/deploy/process-deployment.ts#L31-L32